### PR TITLE
update fluent bit output plugin document

### PR DESCRIPTION
**Pull request by gunnalahema**
Rewriting fluentbit output plugin documentation as it is ambiguous .

fixs [help-request](https://newrelic.slack.com/archives/CEQGKUQ80/p1725455333556649)

### CODE REVIEW

**Comment by nbaenam** [comment]
Hi @gunnalahema, thank you so much for the update. I did some major style edits and mdx cleanup. Let me know what you think!

**Review by nbaenam — APPROVED** [review]

**Comment by gunnalahema** [comment]
Please dont merge it , until we get approval from my team members. Thank you!!

**Comment by nbaenam** [comment]
This revert has been created: https://github.com/newrelic/docs-website/pull/19172

**Pull request merged**
